src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
```

Comment:
don't reset margins for `*`. better to reset special elements that you use

Reply:
checklist
<img width="886" alt="image" src="https://user-images.githubusercontent.com/102411130/218402791-01411485-a239-4c35-8cb7-2c0d0e57f551.png">